src/app/components/cds/inputs/ToggleButtonGroup.js

```diff
@@ -85,6 +89,7 @@ ToggleButtonGroup.propTypes = {
   className: PropTypes.string,
   size: PropTypes.oneOf(['default', 'small', 'large']),
   helpContent: PropTypes.node,
+  iconCenter: PropTypes.bool,
   label: PropTypes.node,
   theme: PropTypes.oneOf(['default', 'setting']),
   variant: PropTypes.oneOf(['contained', 'outlined']),
```

Comment:
@sarahkeh just noticed, shouldn't this include `containedLight`?

Reply:
Good catch, updated!